packages/chakra-ui-core/src/CAccordion/CAccordion.js

```diff
@@ -275,17 +282,17 @@ const CAccordionHeader = {
         py: 2,
         _focus: { boxShadow: 'outline' },
         _hover: { bg: 'blackAlpha.50' },
-        _disabled: { opacity: '0.4', cursor: 'not-allowed' }
-      },
-      attrs: {
+        _disabled: { opacity: '0.4', cursor: 'not-allowed' },
         id: headerId,
         type: 'button',
         disabled: isDisabled,
         'aria-disabled': isDisabled,
-        'aria-expanded': isExpanded,
+        'aria-expanded': isExpanded ? 'true' : 'false',
```

Comment:
this should be string `'true'` & `'false'` or boelean? I wanted to ask u about this

Reply:
The reason I made this a string, was because when the value is a Boolean `true` Vue won't include `aria-expanded="false"` to the element. It will only show the `aria-expanded="true"` when the value is Boolean `true`. But when it's false, it doesn't include the element it on the element.

Making it a string gave me the desired result here.

This is the accordion spec:
![image](https://user-images.githubusercontent.com/21237954/87680944-9d923e00-c7b0-11ea-91d7-c0fdb4a478da.png)

https://www.w3.org/TR/wai-aria-practices-1.1/#accordion

Reply:
i had the same problem somewhere i dont remember where it was but i did exactly this. So thats good.